### ISSUE TITLE
why the test  result of the compressed  gaugan trained by me is not as expected? but  the training result of supernet is right.

### ISSUE THREAD

**Issue by yuanlunxi** [mit-han-lab/gan-compression#71]
_
![下载 (2)](https://user-images.githubusercontent.com/32082617/112090084-dbf8bc00-8bcd-11eb-9d90-20f8bd71e588.png)

**Comment by lmxyy** [comment]
Sorry, I do not quite understand your problem? Could you please elaborate more. Which dataset are you using?

**Comment by yuanlunxi** [comment]
I used the  dataset of myself,the dataset is correct, because the result of training process is correct.

**Comment by yuanlunxi** [comment]
in the fast gan-compression process, when i export the compressed the model from the supernet,should i use the super_mobile_sapde or mobile_spade?

**Comment by yuanlunxi** [comment]
the test output of  head0 of compressed gaugan  is very huge,
![image](https://user-images.githubusercontent.com/32082617/112110219-8f24dd80-8bed-11eb-9c25-ec20f04427df.png)

**Comment by yuanlunxi** [comment]
so is there wrong step cause by me in the export process?

**Comment by dearkafka** [comment]
I bet there is a problem with input data normalization in your code (preprocessing of your dataset for compression)

**Comment by yuanlunxi** [comment]
> I bet there is a problem with input data normalization in your code (preprocessing of your dataset for compression)

the input is normalized

**Comment by yuanlunxi** [comment]
I find the output of 
![image](https://user-images.githubusercontent.com/32082617/112112978-30616300-8bf1-11eb-95d0-889260894951.png)
 is very huge

**Comment by yuanlunxi** [comment]
i find the problem, when the self.training in the SuperSynchronizedBatchNorm2d is set  True  ,the result of test is right. but can anyone explain this ?

**Comment by seekingdeep** [comment]
@yuanlunxi 
- what was the size of each image in your dataset? 256*256?
- are all the images the same size?
- what values did you change to solve this issue
